input/postgres/buffercache.go

```diff
@@ -23,6 +23,7 @@ SELECT block_count * pg_catalog.current_setting('block_size')::int, d.datname, n
   JOIN pg_catalog.pg_database d ON (d.oid = reldatabase)
        LEFT JOIN pg_catalog.pg_class c ON (b.relfilenode = pg_catalog.pg_relation_filenode(c.oid) AND (b.reldatabase = 0 OR d.datname = pg_catalog.current_database()))
        LEFT JOIN pg_catalog.pg_namespace n ON (n.oid = c.relnamespace)
+ WHERE ($1 = '' OR (coalesce(n.nspname, '') || '.' || coalesce(c.relname, '')) !~* $1)
```

Comment:
Generally I would uppercase `COALESCE`, since thats how its done in the Postgres docs: https://www.postgresql.org/docs/12/functions-conditional.html#FUNCTIONS-COALESCE-NVL-IFNULL